source/features/mark-merge-commits-in-list.tsx

```diff
@@ -34,7 +34,7 @@ const filterMergeCommits = async (commits: string[]): Promise<string[]> => {
 
 // eslint-disable-next-line import/prefer-default-export
 export function getCommitHash(commit: HTMLElement): string {
-	return commit.querySelector('a[href]')!.pathname.split('/').pop()!;
+	return commit.querySelector('a[href]:not(.js-issue-link)')!.pathname.split('/').pop()!;
```

Comment:
Does not work? PRs also have the same class

<img width="373" alt="Screen Shot" src="https://user-images.githubusercontent.com/1402241/121506113-a36ac080-ca0d-11eb-831c-ca1036189116.png">

Reply:
PR's? We are catching the wrong link.

From now on I will open an issue for each and thing I want to fix.

Reply:
See https://github.com/pixiebrix/pixiebrix-extension/commits/main

Reply:
> From now on I will open an issue for each and thing I want to fix.

If you're writing a PR, you can include the details in the PR itself to save some time